### PR TITLE
Update ssw-rewards--sprint-forecast-template.md

### PR DESCRIPTION
**Pull request by lukecookssw**
Change Matt G's GitHub handle to the correct one.
Switched Matt G to the primary recipient, with Adam being Cc'd
Removed DevOps link and replaced with GitHub link

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/ssw-rewards--sprint-forecast-template.md [hunk 1]
```diff
@@ -13,30 +13,29 @@ assignees: ''
...
 ### Hi Adam, 
 
 Current Sprint: | ✏️X
 -- | --
-Sprint Goal: | ✏️Content Migration
+Sprint Goal: | ✏️
 Sprint Duration: | ✏️1 week
 Project: | SSW.Rewards
 Project Portal: | https://ssw.visualstudio.com/SSW.Rewards
 Product Owner: | Adam Cogan
```
Comment (matt-goldman) [inline .github/ISSUE_TEMPLATE/ssw-rewards--sprint-forecast-template.md:28]: Product owner has not been changed here

**Review by matt-goldman — CHANGES_REQUESTED** [review]
Update product owner name in template

File: .github/ISSUE_TEMPLATE/ssw-rewards--sprint-forecast-template.md (hunk 1 above)
Comment (lukecookssw) [reply .github/ISSUE_TEMPLATE/ssw-rewards--sprint-forecast-template.md:28]: But, you are not the product owner.

Do you still want this changed?

Comment (matt-goldman) [reply .github/ISSUE_TEMPLATE/ssw-rewards--sprint-forecast-template.md:28]: No, leave it as is

File: .github/ISSUE_TEMPLATE/ssw-rewards--sprint-forecast-template.md [hunk 2]
```diff
@@ -13,30 +13,29 @@ assignees: ''
 [//]: # ( 3. Submit the Issue to send the Sprint Forecast )
 [//]: # (-------------------------------------------------)
 
-To: @adamcogan 
-Cc: @MattGoldmanSSW, @AndreasLengkeek
+To: @matt-goldman 
+Cc: @adamcogan, @JackDevAU
```
Comment (JackDevAU) [inline .github/ISSUE_TEMPLATE/ssw-rewards--sprint-forecast-template.md:17]:
```suggestion
Cc: @adamcogan, @JackDevAU, @lukecookssw 
```

Add yourself :)

File: .github/ISSUE_TEMPLATE/ssw-rewards--sprint-forecast-template.md [hunk 3]
```diff
@@ -13,30 +13,29 @@ assignees: ''
 [//]: # ( 3. Submit the Issue to send the Sprint Forecast )
 [//]: # (-------------------------------------------------)
 
-To: @adamcogan 
-Cc: @MattGoldmanSSW, @AndreasLengkeek
+To: @matt-goldman
```
Comment (JackDevAU) [inline .github/ISSUE_TEMPLATE/ssw-rewards--sprint-forecast-template.md:16]: Is this going back to Adam again?

**Comment by tkapa** [comment]
I'm closing this in favour of #724

**Pull request closed without merging**